Source/WebCore/platform/network/ResourceErrorBase.h

```diff
@@ -58,6 +58,11 @@ class ResourceErrorBase {
     };
     enum class IsSanitized : bool { No, Yes };
 
+    enum class ErrorRecoveryMethod : uint8_t {
```

Comment:
`: bool` ?

Reply:
I used `uint8_t` for extensibility, but you're right that we can just make this a bool right now.